mne/preprocessing/nirs/_beer_lambert_law.py

```diff
@@ -18,15 +18,15 @@
 from ..nirs import _validate_nirs_info, source_detector_distances
 
 
-def beer_lambert_law(raw, ppf=6.0):
+def beer_lambert_law(raw, ppf=[6.0, 6.0]):
     r"""Convert NIRS optical density data to haemoglobin concentration.
 
     Parameters
     ----------
     raw : instance of Raw
         The optical density data.
     ppf : float
```

Comment:
this type is now wrong.

Reply:
Hi, thank you for the hint, but I do not understand. Which type do you mean? Sorry, I am new to mne and python.

Reply:
ppf is not a float anymore here. You make it a list and actually it should be a tuple

Reply:
I see. Thank you for the explanation! I changed it.
Regarding the test for this function, can I just modify the test script to include ppf=(0.121, 0.121) or do I have to generate a new version of [this test data set](https://github.com/mne-tools/mne-testing-data/blob/master/NIRx/nirscout/validation/nirx_15_0_recording_bl.mat) with different ppf's for each wavelength in homer3?